pkg/helm/schema/helm.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://schema.porter.sh/mixins/helm.json",
```

Comment:
This is the new id pattern that I'm suggesting for our mixins

`http://schema.porter.sh/mixins/NAME.json`

eventually our builds will host the mixin schema there, to improve compatibility with most online json schema validators since they are pretty awful and even when these are relative files or just names, like "helm", it tries to resolve it to a file.